### ISSUE TITLE
[Question] SSH into host machine where wireguard is running

### ISSUE THREAD

**Issue by TaridaGeorge** [linuxserver/docker-wireguard#123]
Is there a possibility to ssh into the host machine that hosts the docker where wireguard is running?

My network of PC's look something like this:

PC A, that run's a distribution of Linux (mint), has docker installed in it and this image running.
PC B, with Windows, has wireguard client installed and is connected to the wireguard vpn exposed by PC A.

Normally if PC A and PC B were in the same network, PC B would be able to SSH into PC A, but because the wireguard server runs inside docker I don't know to what ip address I should ssh into.

**Comment by aptalca** [comment]
You would use the lan ip of the server hosting docker

**Comment by TaridaGeorge** [comment]
PC A (the wireguard server) has 192.168.100.0/24 as main lan network. Then there's the docker interface which is 172.17.0.0/24.
PC B is on the lan given by the wireguard server which is 10.13.13.0/24. I've tried both `ssh 192.168.100.100` and `ssh 172.17.0.1` and it didn't work.

**Comment by DerRockWolf** [comment]
Can you reach `192.168.100.100`? Have you tried to ping PC A?

**Comment by TaridaGeorge** [comment]
I cannot reach 192.168.100.100. I have tried pinging PC A but got no response.

**Comment by DerRockWolf** [comment]
You should be able to reach all devices of your network. It would be helpful if you could provide more information about your network, docker settings, etc.

**Comment by aptalca** [comment]
Not enough info provided

**Comment by capntrips** [comment]
In case anyone else ends up here, the host on the default docker bridge network is 172.17.0.1. In my case, I also had docker running on the client machine, so 172.17.0.1 was already in use locally. I moved the Wireguard container to a second bridge network on 172.20.0.0/28 and set my wg0 interface to 172.19.0.0/28, so my configuration below reflects that.

`/config/wg0.conf` in the docker container:

```ini
[Interface]
PrivateKey = [...]
Address = 172.19.0.1/28
ListenPort = 51820
PostUp   = iptables -A FORWARD -i wg0 -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
PostDown = iptables -D FORWARD -i wg0 -j ACCEPT; iptables -t nat -D POSTROUTING -o eth0 -j MASQUERADE

[Peer]
PublicKey = [...]
AllowedIPs = 172.19.0.2/32
```

`/etc/wireguard/wg0.conf` on the client machine:

```ini
[Interface]
PrivateKey = [...]
Address = 172.19.0.2/28

[Peer]
PublicKey = [...]
Endpoint = [host]:51820
AllowedIPs = 172.19.0.1/32
AllowedIPs = 172.20.0.1/32
```

I'm able to ssh into the docker container's host machine from the client machine with `ssh 172.20.0.1`.

tl;dr You probably need to add an `AllowedIP` for the bridge network's gateway to the `Peer` section of your client's config. Also watch out for the default already being assigned if you're also running docker on the client machine.
